src/data-loader.ts

```
@@ -756,53 +875,74 @@ export async function loadDailyUsageData(
+	}
 
-				// Check for duplicate message + request ID combination
-				const uniqueHash = createUniqueHash(data);
-				if (isDuplicateEntry(uniqueHash, processedHashes)) {
-					// Skip duplicate message
-					continue;
-				}
+	const filePromises = sortedFiles.map(async (file) => {
+		const fileEntries = await fileCache.getOrProcess(file, async (file, _baseDir) => {
+			const entries: typeof allEntries = [];
+			const content = await readFile(file, 'utf-8');
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)